versioned_docs/version-2/1-click-signup/api-reference/types.mdx

```diff
@@ -177,7 +237,7 @@ See also our [Postman docs](https://api.docs.verified.inc/).
         [identifierKey: string]: string
     },
     "credentials": {
-        [credentialKey: string]: string
+        [credentialKey: string]: string | [string] | Object | [Object]
```

Comment:
Does `string` and `string[]` really apply? I'm not sure.

Reply:
Yes

Reply:
Detailed explanation in this update (see preview)